### ISSUE TITLE
embedding.optimize() does not have default negative_gradient_method

### ISSUE THREAD

**Issue by dkobak** [pavlin-policar/openTSNE#68]
I tried to reproduce

```
Z = TSNE(perplexity=30, n_jobs=-1, random_state=42, learning_rate=X.shape[0]/12).fit(X)
```

with the step-by-step commands and managed to get the same result with 

```
aff = affinity.PerplexityBasedNN(X, perplexity=30, n_jobs=-1, random_state=42)
init = initialization.pca(X, random_state=42)
embedding = TSNEEmbedding(init, aff, n_jobs=-1, learning_rate=X.shape[0]/12)
embedding = embedding.optimize(n_iter=250, negative_gradient_method='fft', exaggeration=12, momentum=0.5)
embedding = embedding.optimize(n_iter=750, negative_gradient_method='fft', momentum=0.8)

```
however the latter code would not work without `negative_gradient_method='fft'` even though the former code ran fine without explicitly specifying the FFT method. Shouldn't `optimize` have the same default as `TSNE`?

**Comment by dkobak** [comment]
Update: I realized that one can put `negative_gradient_method='fft'` into `TSNEEmbedding()` call and then it's not necessary in `optimize()`. So perhaps it should rather be a default in `TSNEEmbedding()` and not in `optimize`.

**Comment by pavlin-policar** [comment]
Yes, I noticed this a little while back and it is annoying. It should definitely be set to "fft" by default when initializing `TSNEEmbedding`

**Comment by pavlin-policar** [comment]
Fixed in #70
